lib/management/configuration.js

```diff
@@ -260,7 +263,14 @@ function saveConfigurationVersion(cachedOverlay, remoteOverlay, log, cb) {
                 }
                 metadata.putObjectMD(managementDatabaseName,
                     latestOverlayVersionKey, remoteOverlay.version, {}, log,
-                    error => cb(error, remoteOverlay));
+                    error => {
+                        if (!error) {
```

Comment:
We might want to add a `log.error` here.
Btw, I think that the `log.error` message (line 261) is not at the right place since it looks like we are saving the configuration and not its version.